libr/socket/run.c

```diff
@@ -355,6 +355,62 @@ R_API int r_run_parsefile (RRunProfile *p, const char *b) {
 	return 0;
 }
 
+static struct {
+	const char *name;
+	int code;
+} signals[] = {
+	// hardcoded from linux
+	{ "SIGHUP", SIGHUP },
+	{ "SIGINT", SIGINT },
+	{ "SIGQUIT", SIGQUIT },
+	{ "SIGILL", SIGILL },
+	{ "SIGTRAP", SIGTRAP },
+	{ "SIGABRT", SIGABRT },
+	{ "SIGBUS", SIGBUS },
+	{ "SIGFPE", SIGFPE },
+	{ "SIGKILL", SIGKILL },
+	{ "SIGUSR1", SIGUSR1 },
+	{ "SIGSEGV", SIGSEGV },
+	{ "SIGUSR2", SIGUSR2 },
+	{ "SIGPIPE", SIGPIPE },
+	{ "SIGALRM", SIGALRM },
+	{ "SIGTERM", SIGTERM },
+	{ "SIGSTKFLT", SIGSTKFLT },
+	{ "SIGCHLD", SIGCHLD },
+	{ "SIGCONT", SIGCONT },
+	{ "SIGSTOP", SIGSTOP },
+	{ "SIGTSTP", SIGTSTP },
+	{ "SIGTTIN", SIGTTIN },
+	{ "SIGTTOU", SIGTTOU },
+	{ "SIGURG", SIGURG },
+	{ "SIGXCPU", SIGXCPU },
+	{ "SIGXFSZ", SIGXFSZ },
+	{ "SIGVTALRM", SIGVTALRM },
+	{ "SIGPROF", SIGPROF },
+	{ "SIGWINCH", SIGWINCH },
+	{ "SIGIO", SIGIO },
+	{ "SIGPOLL", SIGPOLL },
+	{ "SIGPWR", SIGPWR },
+	{ "SIGSYS", SIGSYS },
+	{ "SIGSTKSZ", SIGSTKSZ },
+	{ NULL }
+};
+
+static int get_signal (const char *e) {
```

Comment:
This is already done in libr/debug/signal.c but maybe makes more sense to have it in libr/util/sys.c